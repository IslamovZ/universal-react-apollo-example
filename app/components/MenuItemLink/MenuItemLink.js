import React, { PropTypes } from 'react';
import { Link } from 'react-scroll';

const MenuItemLink = ({ title }) => (
  <Link to={title.toLowerCase()} smooth>{title}</Link>
);

MenuItemLink.propTypes = {
  title: PropTypes.string.isRequired,
};

export default MenuItemLink;
