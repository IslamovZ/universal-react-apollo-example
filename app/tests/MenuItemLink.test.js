import React from 'react';
import renderer from 'react-test-renderer';

import MenuItemLink from 'components/MenuItemLink/MenuItemLink';

describe('MenuItemLink Component', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <MenuItemLink title="title" />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
