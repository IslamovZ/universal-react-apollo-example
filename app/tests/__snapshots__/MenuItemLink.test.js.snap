exports[`MenuItemLink Component renders correctly 1`] = `
<a
  className={undefined}
  onClick={[Function]}>
  title
</a>
`;
